deployment/localstack/terraform/provider.tf

```diff
@@ -6,3 +6,17 @@ terraform {
     }
   }
 }
+
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+
+  access_key = "fake"
```

Comment:
Do we really need `access_key` and `secret_key`?

Reply:
I don't know it yet. I have some issues on localstack and created a ticket with minimal description. Will look into it after completing this part.
https://github.com/CraftsmenLtd/BloodConnect/issues/123